packages/unity-bootstrap-theme/package.json

```diff
@@ -80,8 +81,7 @@
     "@fortawesome/fontawesome-free": "^5.15.3"
   },
   "volta": {
-    "node": "14.17.6",
-    "yarn": "1.19.1"
```

Comment:
I ran into an issue when I tried to install dependencies using `yarn 1.22.10`, for that I used `yarn 1.19.0`
We can leave it like this if everything works, but we should keep this in mind if we start having problems installing dependencies